.github/workflows/README.md

```diff
@@ -42,7 +42,7 @@
 
 | workflow file                          | action                                                                                                                                                                                                                                           |
 | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| .github/workflows/ci_dockers.yml       | Build docker images used for testing in CI without pushing to the [Docker Hub](https://hub.docker.com/r/pytorchlightning/pytorch_lightning). Publishing these built images takes place in `.github/release-docker.yml` which only runs in mater. |
+| .github/workflows/ci_dockers.yml       | Build docker images used for testing in CI without pushing to the [Docker Hub](https://hub.docker.com/r/pytorchlightning/pytorch_lightning). Publishing these built images takes place in `.github/release-docker.yml` which only runs in master. |
```

Comment:
let's call it the main branch

Reply:
are you suggesting renaming the default branch?
cc: @carmocca

Reply:
Hi @awaelchli - should we let it be `master` as of now so that the reader doesn't assume that our default branch is called `main`? We can reflect the change in all the docs later in case we plan to rename `master` branch to `main` branch. 

Or do you suggest that we refer the `master` branch as `main` branch as it is anyway going to be renamed to `main` soon enough?

Reply:
> should we let it be master for now so that the reader doesn't assume that our default branch is called main? We can reflect the change in all the docs later in case we plan to rename master branch to main branch.

I prefer this option to avoid confusion